### ISSUE TITLE
[BUG] Get-PnPSiteTemplate : The property or field 'ServerRelativePath' has not been initialized.

### ISSUE THREAD

**Issue by frameset** [pnp/powershell#211]
### Reporting an Issue or Missing Feature
Attempting to `get-pnpsitetemplate` fails when using any of the `-Persist*` options.

### Expected behavior 
The template to be written to disk.

### Actual behavior 
Error is thrown as follows:
```
Get-PnPSiteTemplate : The property or field 'ServerRelativePath' has not been initialized. It has not been requested
or the request has not been executed. It may need to be explicitly requested.
At line:1 char:1
+ Get-PnPSiteTemplate -out "assocsite.xml" -PersistBrandingFiles -P ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (:) [Get-PnPSiteTemplate], PropertyOrFieldNotInitializedException
    + FullyQualifiedErrorId : EXCEPTION,PnP.PowerShell.Commands.Provisioning.Site.GetSiteTemplate
```

### Steps to reproduce behavior
```powershell
Connect-PnPOnline -Url "$rooturl/$suburl" -CertificatePath .\PnPPowershellMWLT.pfx -Tenant $tenant -ClientId $id
Get-PnPSiteTemplate -out "assocsite.xml" -PersistBrandingFiles -PersistPublishingFiles -IncludeAllPages
```
At this point the error is thrown.

### What is the version of the Cmdlet module you are running?
(you can retrieve this by executing ```Get-Module -Name "PnP.PowerShell" -ListAvailable```)
1.2.0      PnP.PowerShell
### Which operating system/environment are you running PnP PowerShell on?
- [X] Windows

### Additional - output of `Get-PnPException`
```
Message          : The property or field 'ServerRelativePath' has not been initialized. It has not been requested or
                   the request has not been executed. It may need to be explicitly requested.
Stacktrace       :    at Microsoft.SharePoint.Client.ClientObject.CheckUninitializedProperty(String propName)
                      at Microsoft.SharePoint.Client.File.get_ServerRelativePath()
                      at PnP.Framework.Provisioning.ObjectHandlers.ObjectSiteFooterSettings.PersistFile(Web web,
                   ProvisioningTemplateCreationInformation creationInfo, PnPMonitoredScope scope, String
                   serverRelativeUrl)
                      at PnP.Framework.Provisioning.ObjectHandlers.ObjectSiteFooterSettings.ExtractObjects(Web web,
                   ProvisioningTemplate template, ProvisioningTemplateCreationInformation creationInfo)
                      at PnP.Framework.Provisioning.ObjectHandlers.SiteToTemplateConversion.GetRemoteTemplate(Web web,
                   ProvisioningTemplateCreationInformation creationInfo)
                      at PnP.PowerShell.Commands.Provisioning.Site.GetSiteTemplate.ExtractTemplate(XMLPnPSchemaVersion
                   schema, String path, String packageName, ExtractConfiguration configuration)
                      at PnP.PowerShell.Commands.Provisioning.Site.GetSiteTemplate.ExecuteCmdlet()
                      at PnP.PowerShell.Commands.PnPSharePointCmdlet.ProcessRecord()
ScriptLineNumber : 1
```

**Comment by erwinvanhunen** [comment]
@gautamdsheth created a PR with a fix for this that we just merged. The fix will be available in release 1.2.13-nightly tomorrow.

**Comment by frameset** [comment]
Thanks, just tested it myself and it's working. Hope this makes it to a stable release soon, we hope to use it in a production system!
